test/functional/wallet_bumpfee.py

```diff
@@ -30,14 +30,20 @@
 WALLET_PASSPHRASE = "test"
 WALLET_PASSPHRASE_TIMEOUT = 3600
 
+# Fee rates
+INSUFFICIENT = 0.00001000
+ECONOMY      = 0.00050000
```

Comment:
μNit (ignore unless you rebase or do further updates): `ECONOMICAL`?

Reply:
Good catch! `git grep -ni econom` shows the codebase and `sendtoaddress` rpc help all use "economical".

Reply:
Good catch! git grep -ni econom shows the codebase and rpc help use that. Done.